### DISCUSSION TITLE
Multiple searchurls

### DISCUSSION THREAD

**Discussion by Wolfbeta** [tridactyl/tridactyl#3850] [Ideas]
Is there a way to combine multiple urls to one searchurl to have multiple tabs open?
Something like:
` set searchurls.xy https://x.com/q=%s https://x.com/q=%s`
or:
`set searchurls.x https://x.com/q=%s`
`set searchurls.y https://y.com/q=%s`
`set searchurls.xy searchurls.x searchurls.y %s`

**Comment by bovine3dom** [comment]
Sorry, I thought I'd replied to this. GitHub hides the discussions quite well :)

No, not really, but you can make your own command with e.g.

```
:command googleandbing js -d¾ tri.excmds.tabopen("-b", "bing", JS_ARGS.join(" ")); tri.excmds.tabopen("google", JS_ARGS.join(" "))¾
:googleandbing is google or bing better? # opens the query in bing and google
```
